packages/cozy-client/src/CozyClient.js

```diff
@@ -119,15 +119,15 @@ export default class CozyClient {
     ]
   }
 
-  static registerHook (doctype, name, fn) {
+  static registerHook(doctype, name, fn) {
     CozyClient.hooks = CozyClient.hooks || {}
-    const hooks = CozyClient.hooks[doctype] = CozyClient.hooks[doctype] || {}
+    const hooks = (CozyClient.hooks[doctype] = CozyClient.hooks[doctype] || {})
```

Comment:
That looks like a very hard to find bug.

Reply:
No, it's only a Prettier reformatting, I should add a hook to avoid this

Reply:
Chaining variable assignments is discouraged by airbnb/javascript code style: https://github.com/airbnb/javascript#variables--no-chain-assignment